Utilities/DataLoading/EdFi.SmokeTest.Console/Application/SmokeTestsConfiguration.cs

```diff
@@ -132,6 +136,7 @@ public static SmokeTestsConfiguration Create(IConfigurationRoot configuration)
             {
                 ApiUrl = ResolvedUrl(configuration.GetValue<string>("OdsApi:ApiUrl")),
                 MetadataUrl = ResolvedUrl(configuration.GetValue<string>("OdsApi:MetadataUrl")),
+                XsdMetadataUrl = ResolvedUrl(configuration.GetValue<string>("OdsApi:XsdMetadataUrl")),
```

Comment:
Also initialize the InstanceId property (similar to what you did in BulkLoadClientConfiguration.cs).

Reply:
Code changed as requested.